packages/components/src/components/my-position/my-position.tsx

```diff
@@ -0,0 +1,402 @@
+import { Component, Host, JSX, Prop, Event, EventEmitter, State, h } from '@stencil/core';
+import { UAParser } from 'ua-parser-js';
+import merge from 'deepmerge';
+import midtColors from '@mapsindoors/midt/tokens/color.json';
+import midtOpacity from '@mapsindoors/midt/tokens/opacity.json';
+import { getNestedValue } from '../../utils/utils';
+
+enum PositionStateTypes {
+    POSITION_UNKNOWN = 'POSITION_UNKNOWN',
+    POSITION_REQUESTING = 'POSITION_REQUESTING',
+    POSITION_INACCURATE = 'POSITION_INACCURATE',
+    POSITION_KNOWN = 'POSITION_KNOWN',
+    POSITION_CENTERED = 'POSITION_CENTERED',
+    POSITION_TRACKED = 'POSITION_TRACKED',
+    POSITION_UNTRACKED = 'POSITION_UNTRACKED'
+}
+
+enum DeviceType {
+    Phone = 'mobile',
+    Tablet = 'tablet'
+}
+
+@Component({
+    tag: 'mi-my-position',
+    styleUrl: 'my-position.scss',
+    shadow: true,
+})
+export class MyPositionComponent {
+    @Event({ eventName: 'position_error' }) position_error: EventEmitter<object>;
+    @Event({ eventName: 'position_received' }) position_received: EventEmitter<object>;
+
+    @Prop() mapsindoors;
+    @Prop() myPositionOptions?;
+
+    /**
+     * The current state of device positioning.
+     */
+    @State() positionState: PositionStateTypes;
+
+    private mapView;
+    private options;
+    private compassButton: HTMLButtonElement;
+
+    /**
+     * New UAParser instance.
+     */
+    private parser = new UAParser();
+
+    /**
+     * Wether the MyPositionComponent is a Web Componenet or not.
+     */
+    private readonly isWebComponent: boolean = true;
```

Comment:
This seems unused.

Reply:
@ammapspeople it isn't used here but it is used by the SDK in the BlueDot files: 

<img width="857" alt="Screenshot 2023-07-05 at 09 10 20" src="https://github.com/MapsPeople/web-ui/assets/121795063/830dbc9a-d46f-41cc-824e-cda1acc1d3f7">

Reply:
I'm confused. Are you trying to read a private variable from outside of the component? How does that work?
Anyway, the project cannot be built for production since the linter reports an error about this being unused.

Reply:
Is there another way to determine it, so this property can be removed? 
Can't we just check if the queryselector('mi-my-position') returns an element?

Reply:
@ammapspeople good idea, now it is removed ✅

Reply:
🙌